tests/README.md

```diff
@@ -1,25 +1,239 @@
-Mender Integration testing
-===================
+# Mender Integration Testing
 
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
```

Comment:
I got it, but maybe for those not involved, mention that this is using Emacs/Spacemacs?

Reply:
I'm just gonna remove it actually (Not the TOC, but the marker)

Reply:
Oh, but the comment is useful though! If we wanted to update the TOC in an automated way, I would not know how without this comment.

Reply:
I think I'm gonna leave this one out actually, as it is emacs specific (to a specific package), plus the comment is bullocks, I generate a new one every time.